File: lib/fizz_buzz.rb
class FizzBuzz
  
  def say(valor)
    if (valor % 3 == 0) && (valor % 5 == 0)
      "FizzBuzz"
    elsif (valor % 3 == 0)
      "Fizz"
    elsif (valor % 5 == 0)
      "Buzz"
    else  
      valor.to_s
    end
  end

  
end  

File: dojo_js_start/src/FizzBuzz.js
function FizzBuzz() {
}

FizzBuzz.prototype.say = function(valor) {
  if ((valor % 3 == 0) && (valor % 5 == 0)) {
  	return "FizzBuzz"
  } else if (valor % 3 == 0) {
    return "Fizz"
  } else if (valor % 5 == 0) {
  	return "Buzz"
  } else {
  	return valor;
  } 
};